Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -179,7 +183,6 @@
 		F85F7A2B2655AD5800395515 /* Fuzi in Frameworks */ = {isa = PBXBuildFile; productRef = F85F7A2A2655AD5800395515 /* Fuzi */; };
 		F85F7A2D2655AD5800395515 /* Sentry in Frameworks */ = {isa = PBXBuildFile; productRef = F85F7A2C2655AD5800395515 /* Sentry */; };
 		F85F7A2F2655AD5800395515 /* Telemetry in Frameworks */ = {isa = PBXBuildFile; productRef = F85F7A2E2655AD5800395515 /* Telemetry */; };
-		F85F7A352656885C00395515 /* Glean in Frameworks */ = {isa = PBXBuildFile; productRef = F85F7A342656885C00395515 /* Glean */; };
```

Comment:
Just making sure that the removal of Glean is correct here ... Is Glean now part of Nimbus.framework?

Reply:
Glean is actually bundled as a dependency of the MozillaAppServices swift package -- which means you can still use `import Glean` and all that good stuff as long as you can the Mozilla swift package.